.github/workflows/build-test.yml

```diff
@@ -24,8 +31,8 @@ jobs:
     - name: Check
```

Comment:
Issue: You probably need to install `cargo make` in any CI in which you want to use it.